### DISCUSSION TITLE
ETA for python 3.8 & 3.9 support?

### DISCUSSION THREAD

**Discussion by JohnPreston** [aws-cloudformation/cloudformation-cli-python-plugin#204] [Ideas]
Hello.
Given that python 3.8 and 3.9 are both supported in Lambda for a while now, wondering when we might see it added / supported on `cfn init` and to deploy the resource?
Thank you,

**Comment by kddejong** [comment]
Working on it @JohnPreston.  #198 will add the ability to create hooks and resources in Python 3.8/3.9

This PR in the cloudformation-cli https://github.com/aws-cloudformation/cloudformation-cli/commit/33ef7b9f96245acd380391db8fc55b1b1c3803fb removes the verification of runtimes from the core library so we can just include it in this plugin.

**Reply by JohnPreston** [reply]
Sounds great @kddejong , looking forward to it.

**Reply by carpnick** [reply]
Any kind of ETA?
